### PR TITLE
Feat/dag refactor

### LINKED ISSUE #57: Bug on creating JSON file on Airflow
If the JSON file already exists, the create action fails, making the DAG fail. Create unique names (based on timestamps) on the JSON creation task

### PR DESCRIPTION
**Pull request by hannelita**
This PR fixes #57, #58, #61. It breaks a lot of tests, which I'll check with @shubham-mahajan to see if I can get some help to fix [sorry!!]

Still open problems:
- If triggered with `resource_create` , this DAG cannot find the resource by its ID.
- Still need some refactor for LOGs and for code quality in general
- Working on GCP

- Still need to update README on this project and on connector project

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -22,9 +23,10 @@ pytest-cov==2.9.0         # via -r dev-requirements.in
 pytest-flake8==1.0.6      # via -r dev-requirements.in
 pytest-isort==1.0.0       # via -r dev-requirements.in
 pytest==5.4.3             # via -r dev-requirements.in, pytest-cov, pytest-flake8, pytest-isort
+remote-pdb==2.0.0         # via -r dev-requirements.in
 six==1.15.0               # via packaging, pip-tools
 wcwidth==0.2.4            # via pytest
-mock==4.0.2
```
Comment (shubham-mahajan) [inline dev-requirements.txt:27]: @hannelita You accidentally deleted the mock, one reason for failing tests.

File: aircan/dags/api_ckan_load_gcp.py [hunk 2]
```diff
@@ -2,16 +2,15 @@
 import time
 import json
 import ast
-from datetime import datetime
+from datetime import date, datetime
 
-# DAG Local import
-from aircan.dags.api_ckan_load_multiple_nodes import task_delete_datastore_table, task_create_datastore_table
```
Comment (shubham-mahajan) [inline aircan/dags/api_ckan_load_gcp.py:8]: @hannelita So task_delete_datastore_table, task_create_datastore_table is moved from `dags` to this file? Any specific reason? I think we should keep those in the library.

**Comment by shubham-mahajan** [comment]
@hannelita I am merging this request and start with refactor of tests and other items from the master.

File: aircan/dags/api_ckan_load_gcp.py (hunk 2 above)
Comment (hannelita) [reply aircan/dags/api_ckan_load_gcp.py:8]: @shubham-mahajan I was reusing the same methods from the other DAG (multiple_nodes), but for now I'll rewrite these methods (they are essentially pretty different) and I'll probably delete the other DAG

**Pull request merged**